html5/_main_toc.html.slim

```diff
@@ -213,6 +213,10 @@ nav.sidebar-nav
         li
           a.nav-link href="/latest/plugin_management.html"
             | Plugin Management
+        li
+          a.nav-link href="/latest/aws-tools.html"
+            | Analytic Tools
```

Comment:
Is ` Analytic Tools` really what we want here?

Reply:
Probably not, but I couldn't think of anything better. Do you have a suggestion?

Reply:
How does `AWS Analytics Tools` sound?

Reply:
OK. Changed.

Reply:
OK -- changed.